caffe2/python/workspace_test.py

```diff
@@ -299,10 +299,8 @@ def testTorchInterop(self):
         t.resize_(5)
         t[4] = t[2] = 777
         np.testing.assert_array_equal(t.numpy(), np.array([2,2,777,2,777]))
-        # this doesn't work because of variable / tensor confusion
-        # the underlying data tensor is not properly reshaped :(
         np.testing.assert_array_equal(
-            workspace.FetchBlob("foo"), np.array([2,2,777,2]))
+            workspace.FetchBlob("foo"), np.array([2,2,777,2,777]))
```

Comment:
Does that mean BC? Or just bug fixing?

Reply:
This is technically BC-breaking, but I am not too concerned about it:
1. The test explicitly points out that they wish the resize was reflected in the blob (it just wasn't because of the shallow copy)
2. The BC-breakage is specifically related to PyTorch-Caffe2 interop, which doesn't have too many external users

You could call it BC, you could call it bug fixing.